### PR TITLE
serialver: add page

### PR DESCRIPTION
**Pull request by suhanigarg29**
#9343 
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-   ##[x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by suhanigarg29** [comment]
Could you please merge with Hacktoberfest accepted label, on my final PR. Thanks!

File: pages/linux/serialver.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# serialver
+
+> Returns the serialVersionUID of classes.
+> It does not set a security manager by default.
+> More information: <https://manned.org/serialver.1>.
```
Comment (navarroaxel) [inline pages/linux/serialver.md:5]:
```suggestion
> More information: <https://manned.org/serialver>.
```

File: pages/linux/serialver.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- Display serialVersionUID of a class:
```
Comment (navarroaxel) [inline pages/linux/serialver.md:7]:
```suggestion
- Display the serialVersionUID of a class:
```

File: pages/linux/serialver.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+`serialver {{classnames}}`
+
+- Display the user interface, enter full class name and press show to view the serial version:
```
Comment (navarroaxel) [inline pages/linux/serialver.md:11]: Is this an interactive example?

File: pages/linux/serialver.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+`serialver -show {{classnames}}`
+
+- Display serialVersionUID of a class in a specific package, separate classes and resources with a colon(:):
+
+`serialver -classpath {{directory}} {{classnames}}`
+
+- Pass specified option from reference page of Java application launcher to  the Java Virtual Machine:
```
Comment (navarroaxel) [inline pages/linux/serialver.md:19]:
```suggestion
- Pass specified option from reference page of Java application launcher to the Java Virtual Machine:
```

File: pages/linux/serialver.md (hunk 3 above)
Comment (suhanigarg29) [reply pages/linux/serialver.md:11]: yes

**Comment by suhanigarg29** [comment]
Can this be merged before 31st please, this is my last PR for hacktoberfest.

**Comment by navarroaxel** [comment]
> Can this be merged before 31st please, this is my last PR for hacktoberfest.

Sure! just answer the thread or apply the suggestion.

File: pages/linux/serialver.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+> It does not set a security manager by default.
+> More information: <https://manned.org/serialver.1>.
+
+- Display serialVersionUID of a class:
+
+`serialver {{classnames}}`
+
+- Display the user interface, enter full class name and press show to view the serial version:
+
+`serialver -show {{classnames}}`
```
Comment (navarroaxel) [inline pages/linux/serialver.md:13]: I checked the command and this `-show` flag is not available in the latest version. Just remote it.

File: pages/linux/serialver.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+- Display serialVersionUID of a class in a specific package, separate classes and resources with a colon(:):
+
+`serialver -classpath {{directory}} {{classnames}}`
+
+- Pass specified option from reference page of Java application launcher to the Java Virtual Machine:
```
Comment (navarroaxel) [inline pages/linux/serialver.md:19]:
```suggestion
- Use a specific option from reference page of Java application launcher to the Java Virtual Machine:
```

File: pages/linux/serialver.md (hunk 5 above)
Comment (suhanigarg29) [reply pages/linux/serialver.md]: done

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/serialver.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# serialver
+
+> Returns the serialVersionUID of classes.
+> It does not set a security manager by default.
+> More information: <https://manned.org/serialver>.
```
Comment (kbdharun) [inline pages/linux/serialver.md:5]:
```suggestion
> More information: <https://docs.oracle.com/en/java/javase/18/docs/specs/man/serialver.html>.
```
Update the link to official documentation, also this page must be placed in `common`.

**Review by kbdharun — APPROVED** [review]

File: pages/common/serialver.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+# serialver
+
+> Returns the serialVersionUID of classes.
+> It does not set a security manager by default.
+> More information: <https://docs.oracle.com/en/java/javase/18/docs/specs/man/serialver.html>.
```
Comment (navarroaxel) [inline pages/common/serialver.md:5]:
```suggestion
> More information: <https://docs.oracle.com/en/java/javase/19/docs/specs/man/serialver.html>.
```

**Pull request merged**